### PR TITLE
added feedback bar to top of page.

### PR DESCRIPTION
**Pull request by bottlerocket**
Added the simple feedback bar to the top of the page. I made a new template, and a instance of the headache page to test it.

### CODE REVIEW

**Comment by Demotive** [comment]
Link colour + :hover colour on the yellow background colour don't meet colour contrast requirements...
:visited will pass AA

File: assets/scss/variables/_fs-me.scss [hunk 1]
```diff
@@ -123,3 +124,12 @@ p.lede {
   }
 }
 
+/*-- Feedback bar -- */
```
Comment (Demotive) [inline assets/scss/variables/_fs-me.scss:127]: Should this be in a different scss file?

Comment (bottlerocket) [inline assets/scss/variables/_fs-me.scss:127]: Yes, I'll move it.

**Comment by Demotive** [comment]
Going to close this, the bar is already in the master branch...

**Pull request closed without merging**